source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/ContainerStyleTest.cpp

```diff
@@ -0,0 +1,376 @@
+                        "columns": [
+                            {
+                                "type": "Column",
+                                "items": [
+                                    {
+                                        "type": "TextBlock",
+                                        "text": "No Style"
+                                    }
+                                ]
+                            },
+                            {
+                                "type": "Column",
+                                "style": "default",
+                                "items": [
+                                    {
+                                        "type": "TextBlock",
+                                        "text": "Default Style"
+                                    }
+                                ]
+                            },
+                            {
+                                "type": "Column",
+                                "style": "emphasis",
+                                "items": [
+                                    {
+                                        "type": "TextBlock",
+                                        "text": "Emphasis Style"
+                                    },
+                                    {
+                                        "type": "Container",
+                                        "items": [
+                                            {
+                                                "type": "TextBlock",
+                                                "text": "Container no style"
+                                            }
+                                        ]
+                                    },
+                                    {
+                                        "type": "Container",
+                                        "style": "default",
+                                        "items": [
+                                            {
+                                                "type": "TextBlock",
+                                                "text": "Container default style"
+                                            }
+                                        ]
+                                    },
+                                    {
+                                        "type": "Container",
+                                        "style": "emphasis",
+                                        "items": [
+                                            {
+                                                "type": "TextBlock",
+                                                "text": "Container emphasis style"
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ]
+            }
```

Comment:
We should add this as a sample card :)

Reply:
ok

---
In reply to: [259075079](https://github.com/Microsoft/AdaptiveCards/pull/2406#discussion_r259075079) [](ancestors = 259075079)

Reply:
did I miss this?

---
In reply to: [259093495](https://github.com/Microsoft/AdaptiveCards/pull/2406#discussion_r259093495) [](ancestors = 259093495,259075079)

Reply:
I will add this. I didn't add it here because we don't have a renderer to test it. I thought it would be better to include it when a renderer is added.

---
In reply to: [259540554](https://github.com/Microsoft/AdaptiveCards/pull/2406#discussion_r259540554) [](ancestors = 259540554,259093495,259075079)

Reply:
added the sample card

---
In reply to: [259547086](https://github.com/Microsoft/AdaptiveCards/pull/2406#discussion_r259547086) [](ancestors = 259547086,259540554,259093495,259075079)